apigee_sync_test.go

```diff
@@ -5,90 +5,283 @@ import (
 	"github.com/apigee-labs/transicator/common"
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
+	"net/http/httptest"
+	//"time"
 )
 
-var _ = Describe("listener", func() {
+var _ = Describe("Sync", func() {
 
-	It("should bootstrap from local DB if present", func(done Done) {
+	Context("Sync", func() {
 
-		expectedTables := make(map[string]bool)
-		expectedTables["kms.company"] = true
-		expectedTables["edgex.apid_cluster"] = true
-		expectedTables["edgex.data_scope"] = true
+		var initializeContext = func() {
```

Comment:
If your beforeEach method is declared outside a context, as it currently is, you can only have one.  If you put a beforeEach in each context, then every other context has to have a beforeEach.  Not sure which is actually better, but personally I prefer consolidating the beforeEach, and doing the sync specific stuff locally/manually